settings.py

```diff
@@ -56,18 +47,11 @@
 #    'scraping.middlewares.ScrapingSpiderMiddleware': 543,
 # }
 
-
 # Enable or disable downloader middlewares
 # See https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
-
-# DOWNLOADER_MIDDLEWARES = {
-#         'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware': 110,
-#         'scraping.middlewares.CustomProxyMiddleware': 543,
-# }
-CustomProxyMiddleware = True
 DOWNLOADER_MIDDLEWARES = {
+    'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware': 110,
```

Comment:
This middleware is enabled by default, we do not need to add it here.

Reply:
ok